easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,31 @@
+name = 'TensorFlow'
+version = '1.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
+    'TensorFlow-%(version)s_icc-fixes.patch',
+    'TensorFlow-%(version)s_remove-msse-hardcoding.patch',
+]
+checksums = [
+    '0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8',  # v1.5.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.4.0_swig-env.patch
+    '27f88f61627bf2190e236e350a3651845e8c28de46792c1c5319681a4a4614c7',  # TensorFlow-1.4.0_icc-fixes.patch
+    '71b9d298462f819c5d223c1502305bffc9d29c50e683d04fca8912b59410a142',  # TensorFlow-1.4.0_remove-msse-hardcoding.patch
```

Comment:
@akesandgren Comment is wrong here, patches are named `TensorFlow-1.5.0_*.patch`, just reuse `eb --inject-checksums` to fix?